Parsers/ASimAuthentication/AuthenticationGeneric.yaml

```diff
@@ -0,0 +1,27 @@
+Parser:
+  Title: Empty Authentication Table
```

Comment:
This has the same title as the parser - AuthenticationEmpty.yaml - expected?

Reply:
unexpected